jenkins-design-language/src/js/components/pipeline/PipelineGraph.jsx

```diff
@@ -541,7 +541,7 @@ export class PipelineGraph extends Component {
      */
     renderSelectionHighlight(elements: SVGChildren) {
         const { nodeRadius, connectorStrokeWidth } = this.state.layout;
-        const highlightRadius = nodeRadius + 0.49 * connectorStrokeWidth + 1;
+        const highlightRadius = Math.ceil(nodeRadius + 0.5 * connectorStrokeWidth + 1);
```

Comment:
This was 0.49 to avoid having a slight gap visible IIRC. Have you tested this change on different DPI displays?

Reply:
yup, the gap is now intentional and changing it to 0.5 made the highlight more crips on both my displays (one having a pixel ration of 2 and the other of 1)